internal/services/healthcare/healthcare_dicom_resource.go

```diff
@@ -239,10 +370,38 @@ func resourceHealthcareApisDicomServiceUpdate(d *pluginsdk.ResourceData, meta in
 		return fmt.Errorf("expanding `identity`: %+v", err)
 	}
 
+	var enableDataPartitions *bool
+	if v, ok := d.GetOk("data_partitions_enabled"); ok {
+		enableDataPartitions = pointer.To(v.(bool))
+	}
```

Comment:
This is ForceNew so shouldn't be updateable?

Reply:
Removed from the update function